source/flight_software/propulsion_system.rst

```diff
@@ -0,0 +1,56 @@
+This file describes the Propulsion System driver. 
```

Comment:
In general if this document were more prosaic than bulleted, it would be easier for a new person to understand the propulsion system and our software design strategy around propulsion. I'm not saying getting rid of bullet points entirely is a good idea, but it's important for there to be smooth transitions between software concepts.